build.gradle.kts

```diff
@@ -30,6 +30,7 @@ repositories {
 }
 
 dependencies {
+    implementation("io.pebbletemplates:pebble:3.1.5")
```

Comment:
Use [pebble](https://pebbletemplates.io/) for html templating to avoid building strings dynamically in kotlin.